website/client/src/index.js

```diff
@@ -22,9 +22,11 @@ console.log(store.getState());
 ReactDOM.render(
     <Provider store={store}>
         <BrowserRouter>
+        <HeaderBar />
+        <NavBar />
         <Switch>
-            <Route path="/products" component={Products}/>
-            <Route path="/" component={App}/>
+            <Route key="products" path="/products" component={Products}/>
```

Comment:
What is the purpose of "key" prop here?

Reply:
I forgot erase prop key when I test react-route